typescript-sdk/.gitignore

```diff
@@ -1,2 +1,5 @@
 /dist
-/certs
\ No newline at end of file
+/certs
+
+# ignore docs for now, because we manually use the docs output to construct docs
```

Comment:
you mean we have to copy the output into the fe/docs?

Reply:
no, it's much worse than that. we have to copy over only the `AdminClient.ts` file and the `interfaces` folder, and then change all relative imports and remove some sections from the autogenerated file which aren't useful to the user (like the constructor). I imagine this is all configurable but don't have time to fully automate this.